### ISSUE TITLE
How do you catch the autocomplete.change event?

### ISSUE THREAD

**Issue by Alexandre-T** [afcapel/stimulus-autocomplete#63]
I'm trying to catch the autocomplete.change custom event, but I don't know how to do it.

> autocomplete.change fires when the users selects a new value from the autocomplete field. The event detail contains the value and textValue properties of the selected result.

https://github.com/afcapel/stimulus-autocomplete/blob/019b7ded70b0e3f99ffaf686d508a5b76dc28356/src/autocomplete.mjs#L158

Nothing happen when I tried:
```js
export default class extends Controller {
    change(event) {
        console.log(event);
    }
}
```
During compilation, I have a syntax error on the dot, when I tried:
```js
export default class extends Controller {
    autocomplete.change(event) {
        console.log(event);
    }
}
```
Nothing happen when I tried:
```js
export default class extends Controller {
    'autocomplete.change'(event) {
        console.log(event); 
    }
}

**Comment by Alexandre-T** [comment]
I finally found a solution [on Hotwire](https://discuss.hotwire.dev/t/add-and-remove-eventlisteners/710)!

I share it, it could be useful.

```js
export default class extends Controller {
    connect() {
        document.addEventListener("autocomplete.change", this.autocomplete.bind(this))
    }

    autocomplete(event) {
        console.log(event); 
    }
}
```

If you accept this answer, I could create a pull request and add a few lines in the [README.md file](https://github.com/afcapel/stimulus-autocomplete/blob/019b7ded70b0e3f99ffaf686d508a5b76dc28356/README.md)

**Comment by spaghetticode** [comment]
I think having an example in the readme or a jsfiddle example app would be great for other devs that like me are just starting with Stimulus!

**Comment by Alexandre-T** [comment]
> I think having an example in the readme or a jsfiddle example app would be great for other devs that like me are just starting with Stimulus!

Jobs done! ;-) #65 PR is submitted.

**Comment by Alexandre-T** [comment]
Fixed by #65

**Comment by afcapel** [comment]
@kivanio Listening to an `autocomplete.change` event is not really different than adding an event handler for any other DOM event.

Maybe have a look [at this example](https://github.com/afcapel/stimulus-autocomplete/blob/e311ae8f25d938876b489d93c420bbdb3823c304/examples/events.html#L26-L36), in action at https://stimulus-autocomplete.netlify.app/events.html. 

BTW: in your example, `data-controller="autocomplete new_controller"` should be `data-controller="autocomplete new-controller"`
